CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# How to contribute to this project ?
+
+1. First you must choose a issue on [the project github](https://github.com/mdl29). If you want to do something but no issue were create yet you must create one (try to tag it properly).
+2. Fork the project.
+3. Create a new branch whith the following name format on your fork : `<issueNumber>_<descriptiveNameOfTheIssue>`.
+
+> Example : if you wanted to solve this issue (image below) you could have named your branch `138_Kickstarter_Yanis`.
+> ![example issue](./doc/example_issue.png)
+
+4. When you think your code is ready to be merged to the poject do a pull request, your code will be review and then merged if it's accepted.
+
+# What are the requirement for proposing a pull request ?
+
+## There ares the convention to respect for each programing language
+
+| Language | Convention                             |
+| -------- | -------------------------------------- |
+| Python   | [PEP8](https://pep8.org/)              |
+| YAML     | *(there is a linter but nobody cares)* |
```

Comment:
agree

Reply:
yaml linter is in fact really useful for writing tavern file for unitary test because some scanner module don't really tell why there is an error while yamllint provide all the necessary information.
You should add that it serve for writing unitary test with tavern

Reply:
yaml linter is in fact really useful for writing tavern file for unitary test because some scanner module don't really tell why there is an error while yamllint provide all the necessary information.
You should add that it serve for writing unitary test with tavern